### DISCUSSION TITLE
Performance using large config files

### DISCUSSION THREAD

**Discussion by donnyxray** [XTLS/Xray-core#2515] [Performance]
Is anyone using really large config files?

For example: I currently have approx 280 outbounds in my server side config. I want to expand this to >10.000 outbounds but worry about the performance impact.

**Comment by yomnxkcs** [comment]
I had tested a config.json with 20k outbounds (more than 600k lines) on my computer, and there is no obvious performance issue. That's just my personal test results, yours may be different.
